src/reactviews/pages/QueryResult/resultGrid.tsx

```diff
@@ -125,6 +127,15 @@ const ResultGrid = forwardRef<ResultGridHandle, ResultGridProps>((props: ResultG
         const setupState = async () => {
             await table.setupFilterState();
             await table.restoreColumnWidths();
+            if (context.state.inMemoryDataProcessingThreshold) {
```

Comment:
This should handle the case where a user sets it to a negative number or to 0.  Perhaps both of those are treated as unlimited?

Reply:
I added a minimum value of 1 to the option value, I don't think it makes sense to allow 0 or negative numbers